cmake/projects/FLAC/hunter.cmake

```diff
@@ -0,0 +1,39 @@
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    FLAC
+    VERSION
+    1.3.3-p0
+    URL
+    "https://github.com/cpp-pm/flac/archive/1.3.3-p0.tar.gz"
+    SHA1
+    712ff57b046ed00ebc2898989290c148cfda5c36
+)
+
+set(
+    _hunter_flac_cmake_args
+    WITH_OGG=OFF
+)
+
+if(ANDROID OR IOS)
+  list(
+      APPEND
+      _hunter_flac_cmake_args
+      BUILD_EXECUTABLES=OFF
+      BUILD_EXAMPLES=OFF
+      BUILD_TESTING=OFF
+      BUILD_DOXYGEN=OFF
```

Comment:
Can you move the executables/examples/testing/doxygen to the normal CMake args list?